packages/components/src/common/CloseButton/CloseButton.tsx

```diff
@@ -10,7 +10,7 @@ type CloseButtonProps = {
   /**
    * The color theme of the icon. Also affects the hover state.
    */
-  color: Variant;
+  color?: Variant;
```

Comment:
I noticed that we have a default `color` prop value, but the prop is still required. Made it optional.